ogr/services/github.py

```diff
@@ -13,15 +13,23 @@
 from ogr.abstract import GitUser, PullRequest, PRComment, PRStatus, Release
 from ogr.services.base import BaseGitService, BaseGitProject, BaseGitUser
 from ogr.mock_core import readonly, GitProjectReadOnly
+from ogr.services.mock.github import get_Github_class
 
 logger = logging.getLogger(__name__)
 
 
 class GithubService(BaseGitService):
-    def __init__(self, token=None, read_only=False):
+    def __init__(
+        self, token=None, read_only=False, persistent_storage_file=None, ps_force=False
```

Comment:
It takes me a while to decode the meaning of the `ps_force`.

Please, add at least a doc string for the parameters.

Reply:
I'll rename it to some longer name